components/category/CategoryListItem.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+//Components
+import { IconEdit } from "../Icons";
+import { PTag } from "./../HtmlComponents";
+
+export default function CategoryListItem({ children, id }) {
+  return (
+    <CategoryListItemWrapper>
+      <PTag>{children}</PTag>
+      <Link href={"/edit-entry/" + id} passHref>
+        <EditButton>
+          <IconEdit />
+        </EditButton>
```

Comment:
Please use an anchor Tag here such that the screenreader correctly analise your page. Use 'passHref' instead of href on the Link component. You can style the anchor element that it looks like a button ;)

Reply:
Ah I didn't know that your "Button" is a link, so please change the name to EditAnchor or EditLink